rules/falco_rules.yaml

```diff
@@ -2604,7 +2604,25 @@
   output: "Docker or kubernetes client executed in container (user=%user.name %container.info parent=%proc.pname cmdline=%proc.cmdline)"
   priority: WARNING
   tags: [container, mitre_execution]
+
+
+# This rule is not enabled by default, as there are legitimate use
+# cases for raw packet. If you want to enable it, modify the
+# following macro.
+- macro: consider_raw_packet_communication
+  condition: (never_true)
+
+- list: user_known_raw_packet_binaries
+  items: []
+
+- rule: Raw packet created in container
+  desc: Detect new raw packets at the device driver (OSI Layer 2) level in a container. raw packets could be used to do ARP Spoofing by attacker.
```

Comment:
```suggestion
  desc: Detect new raw packets at the device driver (OSI Layer 2) level in a container. Raw packets could be used to do ARP Spoofing by attacker.
```

Reply:
Thank you for your comment.
I modified desc to use uppercase there.